src/Components/Layout/Header/index.style.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const HeaderWraper = styled.div`
```

Comment:
we should use primereact components, i will do overview

Reply:
Fixed

Reply:
for header i used style components (Amit requirement)